src/components/MoviesList/MoviesList.jsx

```diff
@@ -5,6 +5,6 @@ export const MoviesList = ({ movies }) => (
   <div className="movies">
     {movies.map(movie => (
       <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
+    ))};
```

Comment:
```suggestion
    ))}
```
why you render ; ? check on Demo